libcontainerd/remote/client.go

```diff
@@ -129,17 +127,13 @@ func (c *client) Restore(ctx context.Context, id string, attachStdio libcontaine
 	}, nil
 }
 
-func (c *client) Create(ctx context.Context, id string, ociSpec *specs.Spec, runtimeOptions interface{}, opts ...containerd.NewContainerOpts) error {
+func (c *client) Create(ctx context.Context, id string, ociSpec *specs.Spec, shim string, runtimeOptions interface{}, opts ...containerd.NewContainerOpts) error {
```

Comment:
Wondering if `shim` could be passed as one of the eixisting options (could it be in `opts` or in `runtimeOptions` 🤔

Reply:
It would be nice, but the client seems to do some extra things with the runtime options (which apparently need to be passed on start in addition to create for the v2 runtime?), which need to be passed together.